storage/database/db_manager.go

```diff
@@ -379,13 +383,11 @@ func partitionedDatabaseDBManager(dbc *DBConfig) (*databaseManager, error) {
 			fallthrough
 		case StateTrieDB:
 			newDBC := getDBEntryConfig(dbc, entryType, dir)
-			if dbc.NumStateTriePartitions > 1 {
-				db, err = newPartitionedDB(newDBC, entryType, dbc.NumStateTriePartitions)
-			} else {
-				db, err = newDatabase(newDBC, entryType)
-			}
+			newDBC.DBType = DynamoDB
```

Comment:
@aidan-kwon 
How about change this code in next @winnie-byun 's PR.
I think it better to introduce only dynamoDB implementation.

Reply:
I rebased ..